libkbfs/bare_root_metadata.go

```diff
@@ -933,12 +940,12 @@ func (md *BareRootMetadataV2) GetUnresolvedParticipants() (readers, writers []ke
 
 // GetUserDeviceKeyInfoMaps implements the MutableBareRootMetadata interface for BareRootMetadataV2.
 func (md *BareRootMetadataV2) GetUserDeviceKeyInfoMaps(keyGen KeyGen, _ ExtraMetadata) (
-	readers, writers UserDeviceKeyInfoMap, err error) {
+	readers, writers UserDeviceKeyInfoMap, ok bool, err error) {
```

Comment:
If you implement my other suggestion re. having `GetTLFCryptKeyParams` return an error for v3 if `keyGen < latestKeyGen`, seems more consistent to me to make this function behave similarly instead of returning a bool.